ui/app/templates/components/dashboard/replication-card.hbs

```diff
@@ -0,0 +1,177 @@
+<div class="is-flex-between">
+  <h3 class="title is-4 has-bottom-margin-xxs" data-test-client-count-title>
+    Replication
+  </h3>
+
+  <div>
+    <Hds::Link::Inline @route="vault.cluster.replication.index" class="is-no-underline">Details</Hds::Link::Inline>
+  </div>
+</div>
+
+{{! check if dr replication and performance replication exists }}
+{{#if (or @replication.dr.clusterId @replication.performance.clusterId)}}
+  <hr class="has-background-gray-100" />
+  {{! check if user has access to both perf replication and dr replication }}
+  {{#if (and @version.hasPerfReplication @version.hasDRReplication)}}
+    <div
+      class="is-grid grid-2-columns grid-gap-2 has-top-margin-m has-bottom-margin-m grid-align-items-start"
+      data-test-dr-perf-replication
+    >
+      <div>
+        <StatText
+          @label="state"
+          @value={{@replication.dr.state}}
+          @size="l"
+          @subText="The current operating state of the cluster."
+          data-test-stat-text="dr state"
+        >
+          <:titleLink>
+            <Hds::Link::Inline
```

Comment:
Probably a question for design - but reading [here](https://helios.hashicorp.design/components/link/inline#display-sizes) it seems like this title should not be styled as a link and instead a standalone link should render below?

Reply:
@hellobontempo This is definitely a question for design. When we had the initial design discussions, the thought is to have them as links so users can navigate to that page - it does go beyond our conventions so open to your thoughts here.